packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -11667,11 +11661,8 @@ export class AdapterClass extends EventEmitter {
         try {
             tools.validateGeneralObjectProperties(task, true);
         } catch (e) {
-            // todo: in the future we will not create this object
-            this._logger.warn(`${this.namespaceLog} Object ${task._id} is invalid: ${e.message}`);
-            this._logger.warn(
-                `${this.namespaceLog} This object will not be created in future versions. Please report this to the developer.`
-            );
+            this._logger.error(`${this.namespaceLog} Object ${task._id} is invalid: ${e.message}`);
```

Comment:
maybe we can also keep this, as these are the instanceObjects from ioPack and probably not much should occur here and if so it is easy to fix

Reply:
Yes, I think thats ok ... when dev defines wrong things, it m,ight be ok to dcrash. lets see how it is on alpha